tests/sentry/integrations/slack/test_message_builder.py

```diff
@@ -822,6 +822,89 @@ def test_escape_slack_message(self):
         assert isinstance(ret, dict)
         assert "<https://example.com/|*Click Here*>" in ret["blocks"][1]["text"]["text"]
 
+    @with_feature("organizations:slack-metric-alert-description")
```

Comment:
mind adding a test w/o this to make sure the feature works as expected w/o the flag enabled?